src/http_rest_api_handler.cpp

```diff
@@ -367,13 +374,16 @@ Status HttpRestApiHandler::processModelControlApiRequest(std::string& response)
         return status;
     }
 
-    std::string jsonString;
-    status = GetModelStatusImpl::serializeModelsStatuses2Json(modelsStatuses, jsonString);
+    std::string modelsStatusesJson;
+    status = GetModelStatusImpl::serializeModelsStatuses2Json(modelsStatuses, modelsStatusesJson);
     if (!status.ok()) {
         response = createErrorJsonWithMessage("Serializing model statuses to json failed.");
         return status;
     }
-    response = jsonString;
+
+    std::string pipelinesStatusesJson = manager.getPipelinesStatusesAsJson();
+
+    response = "\"models\" : \n" + modelsStatusesJson + ",\n\"pipelines\" : \n" + pipelinesStatusesJson;
```

Comment:
use std::string += instead. I am wondering if passing the response string reference to the:
-> getPipelinesStatusesAsJson
-> getModelsStatusesAsJson functions wouldn't be better as well.

Alternatively std::stringstream